pom.xml

```diff
@@ -16,13 +16,30 @@
 
         <!-- Dependency versions -->
         <junit.version>5.10.3</junit.version>
+        <guava.version>33.3.1-jre</guava.version>
 
     </properties>
 
     <modules>
         <module>proxy-socket-core</module>
+        <module>proxy-socket-udp</module>
+        <!-- <module>proxy-socket-tcp</module>
+        <module>proxy-socket-guava</module>
+        <module>proxy-socket-examples</module> -->
     </modules>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.testcontainers</groupId>
+                <artifactId>testcontainers-bom</artifactId>
+                <version>2.0.1</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
```

Comment:
I'm not sure if this is really used ? 

But mockito dependency is missing or ?